### DISCUSSION TITLE
How to use camera stream?

### DISCUSSION THREAD

**Discussion by sausagerus** [copystring/ioBroker.roborock#407] [Q&A]
Hello,
where should i paste my password in stream links from my object list rtsp or stream mp4 ?
for example my pass is "PASS"
what is my username? 
rtsp://USERNAME:PASS@192.***.***.157:8554/45SAELs1Ykt3zoa5KPzX0L?video

I would like to send me a picture from my S6 MaxV via telegram when i click a button

Or i trying to use it completly wrong ?

with best regards 
sausagerus

**Comment by copystring** [comment]
Hi, does this help you? https://forum.iobroker.net/topic/62328/roborock-adapter-tester-gesucht/510?_=1697747372505
If set up correctly, you can copy paste your rtsp stream to vlc player to check if it works correctly. No need to change anything in the rtsp stream. Username and PW are generated by the adapter.
